example/src/blocks/image-example/block.json

```diff
@@ -5,13 +5,24 @@
     "description": "Example Block to show the Image in usage",
     "icon": "smiley",
     "category": "common",
-    "example": {},
+    "example": {
+		"attributes": {
+			"image": {
+				"id": -1,
+				"fallback": "https://images.unsplash.com/photo-1666625267725-310b76f139d6"
+			}
+		}
+	},
     "supports": {
         "html": false
     },
     "attributes": {
-        "imageId": {
-            "type": "number"
+        "image": {
+            "type": "object",
+			"default": {
+				"id": null,
+				"fallback": null
+			}
         },
```

Comment:
@fabiankaegy Essentially, I think we could have gone two ways about this:

1. The way you approached it here by using a single prop structured as an object with two values, i.e., `id` & `fallback`, respectively. However, I'm just wondering about any possible confusion about what type of data needs to be provided for these two values, i.e., number vs. string.
2. **OR** perhaps it would be more clear (and less complex) to define the fallback image URL as a separate prop (optional), e.g.,  
```
"imageId": {
    "type": "number",
    "default": null
},
"fallbackImageUrl": {
    "type": "string",
    "default": ""
}